fakestorage/object_test.go

```diff
@@ -604,3 +604,42 @@ func TestServerClientObjectDeleteErrors(t *testing.T) {
 		}
 	})
 }
+
+func TestServerClientObjectSetAclPrivate(t *testing.T) {
+	objs := []Object{
+		{BucketName: "some-bucket", Name: "img/public-to-private.jpg"},
+	}
+
+	runServersTest(t, objs, func(t *testing.T, server *Server) {
+		t.Run("public to private", func(t *testing.T) {
+			ctx := context.Background()
+			objHandle := server.Client().Bucket("some-bucket").Object("img/public-to-private.jpg")
+
+			err := objHandle.ACL().Set(ctx, storage.AllAuthenticatedUsers, storage.RoleReader)
+			if err != nil {
+				t.Errorf("unexpected error while setting acl %+v", err)
+				return
```

Comment:
use ``t.Fatalf`` instead of ``t.Errorf`` + ``return``.

Reply:
done